src/Symfony/Bundle/FrameworkBundle/Resources/config/translation.xml

```diff
@@ -152,5 +152,10 @@
         <service id="translation.extractor" class="%translation.extractor.class%"/>
 
         <service id="translation.writer" class="%translation.writer.class%"/>
+
+        <service id="translation.warmer" class="Symfony\Bundle\FrameworkBundle\CacheWarmer\TranslationsCacheWarmer" public="false">
+            <argument type="service" id="translator" />
```

Comment:
If we use the alias "translator", in dev mode this won't work as  the translator return an instance of [LoggingTranslator](https://github.com/symfony/symfony/blob/2.7/src/Symfony/Component/Translation/LoggingTranslator.php).